### ISSUE TITLE
Frame compression during inference after LoRA fine-tuning of Wan2.2

### ISSUE THREAD

**Issue by james-tensor** [kohya-ss/musubi-tuner#714]
Thank you very much—excellent work!
I’m using this project to fine-tune the Wan2.2 model with LoRA. However, when I run inference with the LoRA model, I’ve noticed that the frame is compressed. Characters, objects, and everything else appear squashed vertically. Below are my training parameters. Could you help me check where the issue might be? 
All the training data are videos with normal frame proportions, and there are no images.
Thank you so much!

lora_config.toml
[general]
resolution = [240,240]  
batch_size = 1  
bucket_no_upscale = false  
[[datasets]]
video_jsonl_file = "/root/src/musubi_tuner/dataset/tan/train_lora_data.jsonl" 
frame_extraction = "full"  
cache_directory = "/root/src/musubi_tuner/dataset/tan/cache"  
num_repeats = 3  

train shell code
accelerate launch --num_cpu_threads_per_process 4 --mixed_precision fp16 src/musubi_tuner/wan_train_network.py --task t2v-A14B --dit /root/pretrained_model/wan2.2_t2v_high_noise_14B_fp16.safetensors/split_files/diffusion_models/wan2.2_t2v_high_noise_14B_fp16.safetensors --dataset_config /root/src/musubi_tuner/dataset/tan/lora_config.toml --mixed_precision fp16 --optimizer_type AdamW --learning_rate 2e-4 --gradient_checkpointing --max_data_loader_n_workers 4 --persistent_data_loader_workers --network_module networks.lora_wan --network_dim 32 --timestep_sampling shift --discrete_flow_shift 5.0 --min_timestep 0 --max_train_epochs 1000 --timestep_boundary 875 --seed 42 --preserve_distribution_shape --output_dir /root/lora_train/tan/t2v/high --output_name tan --save_every_n_steps 100 --flash_attn --split_attn --gradient_accumulation_steps 4 --vae /root/pretrained_model/Wan2.1_VAE.pth --t5 /root/pretrained_model/models_t5_umt5-xxl-enc-bf16.pth --sample_prompts /root/src/musubi_tuner/dataset/tan/test_prompt.txt --sample_every_n_steps 400 --sample_at_first

https://github.com/user-attachments/assets/c79b15a2-da2f-408c-b2ea-79141c5b106a

**Comment by kohya-ss** [comment]
This may be related to #656. Could you please tell us the apparent aspect ratio of the original video and the pixel size of the frame size?

**Comment by james-tensor** [comment]
The pixel size of the training videos I used is 960×544, with an actual aspect ratio of 9:16. Having read the hints in #656, I previously trained LoRA with non-square training data without encountering frame compression issues. Could you please advise why this problem is triggered only this time?

**Comment by kohya-ss** [comment]
Please add the `--debug_mode video` option to your cache script to see the actual data used for training. This option will save it in mp4 format in the cache directory.

If the data is not what you expected, please tell us how it is wrong. This may give us a clue to how to solve the problem.
